core/domain/exp_domain.py

```diff
@@ -588,6 +589,7 @@ def __init__(
                 is created.
             last_updated: datetime.datetime. Date and time when the exploration
                 was last updated.
+            proto_size_in_bytes: int. Size of exploration.
```

Comment:
Give more detail in the description (at least include info about "proto" and "bytes").

Reply:
Done